contrib/epee/include/misc_language.h

```diff
@@ -128,6 +128,59 @@ namespace misc_utils
 
   }
 
+  template<typename T>
```

Comment:
`epee` is typically networking related code and stuff to support the networking code. Based on your description, it will not be used by networking code, so move to `src/` somewhere.

Reply:
I am ok with moving it, but feel uncertain about `is typically networking related code and stuff to support the networking code`.

I chose this file because it already has `median` and `get_mid`.
- `median` is only used in `src` stuff (and tests) as far as I can tell.
- `get_mid` is used in `median` and `rolling_median`
- `rolling_median`, while defined in `epee`, is not related to networking.

Reply:
Possibly, if they aren't used in epee elsewhere.